examples/examples/multi_logger.rs

```diff
@@ -153,6 +153,11 @@ async fn run_server() -> anyhow::Result<SocketAddr> {
 		Ok(())
 	})?;
 	let addr = server.local_addr()?;
-	server.start(module)?;
+	let handle = server.start(module)?;
+
+	// In this example we don't care about doing shutdown so let's it run forever.
+	// You may use the `ServerHandle` to shut it down or manage it yourself.
+	tokio::spawn(async move { handle.await });
```

Comment:
if `ServerHandle` has a method like `stopped` which called `closed` and returns `Future + Send + 'static`, perhaps this can just be `tokio::spawn(handle.stopped())` which isn't too bad :)